app/views/admin/shared/tagging/_breadcrumb_list.mustache

```diff
@@ -1,5 +1,6 @@
 {{#breadcrumbs}}
 <div class="taxon-breadcrumb">
+  <button type="button" class="close deselect-taxon-button" aria-label="Remove"><span aria-hidden="true">&times;</span></button>
```

Comment:
This should be after the ol, it seems weird to be putting the button first in the markup when it technically appears after the list in the page and screenreaders will read the remove button out first before the taxon breadcrumb.

Reply:
Instead of `aria-label="Remove"`, aria-label="Deselect taxon" might be clearer.

Reply:
Sounds sensible, I've changed the ordering here 8fcf69205953214a396aacf708d0cdf33a8aa883

Reply:
and I've change the copy in e3e912858060969759b193b0ce179e9984d1ab36

I went with "Deselect topic" (rather than "Deselect taxon"), as topic is the name used elsewhere on the page.